api/services/packages.ts

```diff
@@ -51,3 +51,12 @@ export const useViewPackage = (packageId: string) => {
         queryKey: [baseQueryKey],
     });
 }
+
+export const useEditPackage = ( editedPackage: PaginatedResponse<Package>, id?: string) => {
```

Comment:
this would throw an error!. the data type you are passing is invalid. why are you sending a paginated response data as a post or patch request?.  patch request should only have payload information associated with the Package. sending a data type that is paginated would alter the data structure